hackathon-topics.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
Please split it up in one file per topic and only combine it back into one during build as otherwise it will result in ongoing merge conflicts due to reformats etc.

Reply:
I think changes is sequential JSON Blocks will only lead to trivial merge problems. I will happily fix them once they appear.
This format has also worked well for larger hackathons.